go.mod

```diff
@@ -5,22 +5,23 @@ go 1.12
 require (
 	github.com/blang/semver v3.6.1+incompatible
 	github.com/go-ldap/ldap v3.0.3+incompatible // indirect
-	github.com/hashicorp/go-hclog v0.9.0 // indirect
+	github.com/hashicorp/go-hclog v0.9.2 // indirect
+	github.com/hashicorp/go-plugin v1.0.1 // indirect
 	github.com/lib/pq v1.1.1 // indirect
-	github.com/mattermost/go-i18n v1.10.0 // indirect
-	github.com/mattermost/mattermost-server v0.0.0-20190516103213-2d3e41783398
-	github.com/nicksnyder/go-i18n v1.10.0 // indirect
+	github.com/mattermost/go-i18n v1.11.0 // indirect
+	github.com/mattermost/mattermost-server v0.0.0-20190624195857-dbe330511eb4
 	github.com/pelletier/go-toml v1.4.0 // indirect
 	github.com/pkg/errors v0.8.1
 	github.com/stretchr/objx v0.2.0 // indirect
 	github.com/stretchr/testify v1.3.0
 	go.uber.org/atomic v1.4.0 // indirect
 	go.uber.org/zap v1.10.0 // indirect
-	golang.org/x/crypto v0.0.0-20190506204251-e1dfcc566284 // indirect
-	golang.org/x/net v0.0.0-20190503192946-f4e77d36d62c // indirect
-	golang.org/x/sys v0.0.0-20190508100423-12bbe5a7a520 // indirect
-	google.golang.org/genproto v0.0.0-20190508193815-b515fa19cec8 // indirect
-	google.golang.org/grpc v1.20.1 // indirect
+	golang.org/x/crypto v0.0.0-20190621222207-cc06ce4a13d4 // indirect
```

Comment:
I'm a bit surprised this indirect dependencies still show up. Should't they be removed?

Reply:
I don't edit these manually I just use `go mod tidy`.

Reply:
Not blocking: Why aren't you deleting these manually? It _seams_ to me that `go mod tidy` should do this, but doesn't due to a bug.

Reply:
I thought the idea of these files is that they are managed by the `go mod` command. I am inclined just to trust whatever `go mod tidy` thinks is right. Kinda like I trust `gofmt` (or really `goimports`).

Reply:
I agree. Let's leave this as it is and see what go1.13 brings.